app/helpers/application_helper.rb

```diff
@@ -2,7 +2,7 @@ module ApplicationHelper
   include NavigationHelper
 
   def format_currency(amount, **options)
-    number_to_currency(amount, options)&.sub(/\.00$/, '')
+    number_to_currency(amount, options.reverse_update(precision: 0))&.sub(/\.00$/, '')
```

Comment:
I think you can set precision in the `en.yml` file.

`&.sub(/\.00$/, '')` is also now redundant.